client/signup/controller.js

```diff
@@ -52,6 +52,12 @@ export default {
 	redirectTests( context, next ) {
 		if ( context.pathname.indexOf( 'new-launch' ) >= 0 ) {
 			next();
+		} else if (
+			context.pathname.indexOf( 'domain' ) >= 0 ||
+			context.pathname.indexOf( 'plan' ) >= 0
```

Comment:
Should we test for URLs with `plan` in the route as well?

Reply:
well, I was sleeping when you commented, and it's already merged but .. yeah :D  (sorry, I updated the PR and forgot to update the instructions).